inlong-sort/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/sink/FlinkSink.java

```diff
@@ -716,7 +716,7 @@ static IcebergProcessOperator<RowData, WriteResult> createStreamWriter(Table tab
                         appendMode);
 
         return new IcebergProcessOperator<>(new IcebergSingleStreamWriter<>(
-                table.name(), taskWriterFactory, inlongMetric, auditHostAndPorts, null, dirtyOptions, dirtySink));
+                table.name(), taskWriterFactory,  null));
```

Comment:
No need two blank before `null`.
Please refer the https://github.com/apache/inlong/actions/runs/4250657643/jobs/7392028444, and fix the format violations, thanks.